COMET.Web.Common/Components/SingleIterationApplicationTemplate.razor.cs

```diff
@@ -136,7 +143,7 @@ internal void SetCorrectUrl()
             {
                 currentOptions[QueryKeys.IterationKey] = this.ViewModel.SelectedIteration.Iid.ToShortGuid();
                 currentOptions[QueryKeys.ModelKey] = this.ViewModel.SelectedIteration.IterationSetup.Container.Iid.ToShortGuid();
-                currentOptions[QueryKeys.ServerKey] = this.ViewModel.SessionService.Session.DataSourceUri;
+				currentOptions[QueryKeys.ServerKey] = this.ViewModel.SessionService.Session.DataSourceUri;
```

Comment:
indentation issue in the file. can you also remove the option from the url if the static address is defined?